commit 09322ef8f4b8fc1f03d1c4d975b1edf2bc4cf5d1 (HEAD -> master)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-06-28T14:05:25-04:00 (3 months ago)
    
    TOOLS-2272 remove NODE_PREBUILT from pure libraries
    Reviewed by: Isaac Davis <isaac.davis@joyent.com>
    Approved by: Rui <rui@joyent.com>

diff --git a/Makefile b/Makefile
index a28afaf..a76af0b 100644
--- a/Makefile
+++ b/Makefile
@@ -22,26 +22,15 @@ TAP := ./node_modules/.bin/tap
 JS_FILES := $(shell find lib -name '*.js')
 ESLINT_FILES := $(JS_FILES)
 
-ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_VERSION =	v4.6.1
-	NODE_PREBUILT_TAG =	zone
-	NODE_PREBUILT_IMAGE =	18b094b0-eb01-11e5-80c1-175dac7ddf02
-endif
-
 include ./tools/mk/Makefile.defs
-ifeq ($(shell uname -s),SunOS)
-	include ./tools/mk/Makefile.node_prebuilt.defs
-else
-	NODE := node
-	NPM := $(shell which npm)
-	NPM_EXEC=$(NPM)
-endif
 include ./tools/mk/Makefile.node_modules.defs
 
 #
 # Variables
 #
 
+NODE := node
+NPM := npm
 TEST_UNIT_JOBS ?= 4
 BUILD = $(TOP)/build
 CLEAN_FILES += $(BUILD)
@@ -103,8 +92,5 @@ git-hooks:
 	ln -sf ../../tools/pre-commit.sh .git/hooks/pre-commit
 
 include ./tools/mk/Makefile.deps
-ifeq ($(shell uname -s),SunOS)
-	include ./tools/mk/Makefile.node_prebuilt.targ
-endif
 include ./tools/mk/Makefile.targ
 include ./tools/mk/Makefile.node_modules.targ
